### ISSUE TITLE
Mentions as WebFinger should not apply to user of local instance.

### ISSUE THREAD

**Issue by thelovebug** [andstatus/andstatus#448]
Right, how to explain this one...

If you choose any of the Reply options to a notice, you get one or more WebFinger URIs prepopulated at the start of your reply, as per this screenshot:

![screenshot_20170318-134341](https://cloud.githubusercontent.com/assets/2915687/24072600/6f0f0740-0be1-11e7-90ea-5e633ef37924.png)

However, if one of the users that are being mentioned in your reply happens to be a member of the instance on which you also reside, then the use of a full WebFinger ID results in the mention not being fully resolved once it is posted, as per this screenshot (reply 3):

![screenshot_20170318-134257](https://cloud.githubusercontent.com/assets/2915687/24072650/1bfb2858-0be2-11e7-8f1b-67cf81c73412.png)

**This is a GNUsocial thing, not an AndStatus thing.**

This is a feature request by which if a mentioned user is a member of the same instance that your account is on (which should be easy to identify from the account name and the WebFinger ID of the user you're mentioning), only their @username should be populated into the reply field.

As per this alternative to the first screenshot:

![screenshot_20170318-135657](https://cloud.githubusercontent.com/assets/2915687/24072710/f00f1064-0be2-11e7-81ee-2096f030af39.png)

**Comment by yvolk** [comment]
Please see https://github.com/andstatus/andstatus/issues/312 - it's about the same.
You may also see my posts at loadaverage.org, which all are done via AndStatus: https://loadaverage.org/andstatus
after server-side processing webfingerIds of mentioned users are transformed into shorter links...

**Comment by thelovebug** [comment]
Forgive me, I did search for this issue before I posted a new one.

Having spoken to the developer of postActiv, apparently this is _not_ standard behaviour within GNUsocial.  EIther there is a setting that I'm missing somewhere, or other instances have made code changes to override this behaviour.  I wonder if someone did make such a change to loadaverage.org.

I'll close this for now and pursue other avenues.  Thanks!
